mifospay/src/main/res/layout/activity_si_details.xml

```diff
@@ -0,0 +1,280 @@
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".standinginstruction.ui.SIDetailsActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <include layout="@layout/toolbar"/>
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <android.support.constraint.ConstraintLayout
+                android:id="@+id/layout_si_details"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <TextView
+                    android:id="@+id/tv_si_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/marginItemsInSectionMedium"
+                    android:textColor="@color/black"
+                    android:textSize="40sp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/value_40dp"
+                    android:layout_marginStart="@dimen/value_20dp"
+                    android:layout_marginEnd="@dimen/value_20dp"
+                    android:layout_marginLeft="@dimen/value_20dp"
+                    app:layout_constraintTop_toBottomOf="@id/tv_si_name"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:textColor="@color/gray_dark"
+                        android:textSize="@dimen/value_16sp"
+                        android:text="Standing Instruction ID:"/>
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:id="@+id/tv_si_id"
+                        android:textColor="@color/colorAccent"
+                        android:textSize="@dimen/value_16sp"
+                        android:layout_marginBottom="@dimen/marginItemsInSectionMedium" />
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:textColor="@color/gray_dark"
+                        android:textSize="@dimen/value_16sp"
+                        android:text="Amount:"/>
```

Comment:
It would be much better if it wasn't hardcoded otherwise will not be able to translate it :)